composer.json

```
@@ -37,7 +37,7 @@
         "drupal/component": "1.0-rc3",
         "drupal/config_ignore": "2.3.0",
         "drupal/config_rewrite": "^1.4",
-        "drupal/core": "~9.2.15 || ~9.3.8",
+        "drupal/core": "~9.2.15 || ~9.3.9",
```

@rajeshreeputra Please update drupal 9.2 version as well i.e
```suggestion
        "drupal/core": "~9.2.16 || ~9.3.9",
```

---

Updated drupal 9.2 version